04_environment/index.tex

```diff
@@ -63,10 +54,7 @@ \subsubsection{Tier System}
     \label{fig:Foraging Tier System}
 \end{figure}
 
-Higher tiers result in a lower cost per animal, however, the total cost increases up to a daily catch limit.
-This means that if teams collaborate, they can invest less as individuals to get a collectively greater return through deer hunting. If they invest too much then they will over spend as they reach the daily catch limit. As a result, it is more beneficial for some teams to go fishing because it has a higher daily catch limit, although it comes with lower returns. In other words, there is a need for some self sacrifice to be done to reach maximum welfare.
-
-Fishing has similar functions with the key difference being the tiers start at $1$ and Normal distribution.
+The amount of resources inputted is scaled using a variable and the tiers are then calculated with them, the higher the tiers the lower the cost to catch the next animal. However, the total cost increases up to a daily catch limit. This means that if teams collaborate, they can invest less as individuals to get a collectively greater return through deer hunting. If they invest too much then they will over spend as they reach the daily catch limit. As a result, it is more beneficial for some teams to go fishing because it has a higher daily catch limit, although it comes with lower returns. In other words, there is a need for some self sacrifice to be done to reach maximum welfare. Fishing has similar functions with the key difference being the tiers start at and its Normal distribution.
```

Comment:
revise the first sentence.
not very understandable :(


do NOT forget the comma:
If they invest too much,

do NOT forget the dash:
**self-sacrifice**

Reply:
to reach **the** maximum welfare.

define welfare? what do you mean by it? footnote should do the work

Reply:
last sentence -- not understandable.
revise it plz